features/topology-map/plugins/org.opennms.features.topology.plugins.topo.pathoutage/src/main/java/org/opennms/features/topology/plugins/topo/pathoutage/PathOutageProvider.java

```diff
@@ -74,11 +79,18 @@ public void refresh() {
 
 	@Override
 	public Defaults getDefaults() {
+		// Current implementation of default focus strategy includes all "problematic" (state not NORMAL) nodes, semantic zoom level is 0
 		return new Defaults()
-				.withSemanticZoomLevel(Defaults.DEFAULT_SEMANTIC_ZOOM_LEVEL)
+				.withSemanticZoomLevel(0)
 				.withPreferredLayout("Hierarchy Layout")
 				.withCriteria(() -> {
-					List<VertexHopGraphProvider.VertexHopCriteria> focusCriteria = FocusStrategy.ALL.getFocusCriteria(PathOutageProvider.this);
+					List<VertexHopGraphProvider.VertexHopCriteria> focusCriteria = new ArrayList<>();
+					for (Vertex vertex : this.getVertices()) {
```

Comment:
The default focus should be the vertex with the worst severity.
At the moment all vertices which have a severity != NORMAL are selected.
In addition if everything is okay, the default focus results in `no vertices` if there are no problems.

That beeing said, wouldn't it be easier to calculate the status of all vertices by invoke `statusProvider.getVertices(...)`, sort them by severity and select the worst one.